condor_watch_q.py

```diff
@@ -988,6 +996,15 @@ def make_table(headers, rows, fill="", header_fmt=None, row_fmt=None, alignment=
         for original_row, processed_row in zip(rows, processed_rows)
     ]
 
+    remaining_columns = terminal_columns - len(lines[0])
+    if progress_bar and remaining_columns > 10:
```

Comment:
I think I might want the per-row progress bars to be a separate option from the total progress bar. It doesn't cost us much and lets people customize as-desired.